webview-ui/src/context/ExtensionStateContext.tsx

```diff
@@ -62,9 +63,7 @@ export interface ExtensionStateContextType extends ExtensionState {
 	setCustomSupportPrompts: (value: CustomSupportPrompts) => void
 	enhancementApiConfigId?: string
 	setEnhancementApiConfigId: (value: string) => void
-	experimentalDiffStrategy: boolean
-	setExperimentalDiffStrategy: (value: boolean) => void
-	autoApprovalEnabled?: boolean
```

Comment:
Is this removal intentional?

Reply:
yes, it's intentional because ExtensionStateContextType extend ExtensionState, and autoApprovalEnabled still have in that ExtensionState, so i clean up a bit